### DISCUSSION TITLE
is there a way to hide cells without using cell meta data?

### DISCUSSION THREAD

**Discussion by murphyk** [executablebooks/meta#740] [Q&A]
Hi

I understand from https://jupyterbook.org/en/stable/content/metadata.html that the way to hide code cells is to insert cell meta data to contain
```
 {
    "tags": [
        "hide-cell"
    ]
}
```
Unfortunately I am developing my notebooks inside VS code, which currently does not support this feature 
(see [this issue](https://github.com/microsoft/vscode-jupyter/issues/1121)). Is there another way within jbook to hide cells,
or do I have to switch my workflow to using jupyter lab (say)?

**Comment by mmcky** [comment]
@murphyk It sounds like you're authoring in `ipynb` notebooks. The project is currently working on tooling to render `myst` in that context via `mystjs` but that project is still in development. We are looking forward to that functionality (cc: @rowanc1 @chrisjsewell)

At [quantecon](https://quantecon.org) our "typical" workflow is to prototype in a `notebook` and then move to use `myst:md` files to make version control easier on GitHub etc. There is a tool [jupytext](https://jupytext.readthedocs.io/en/latest/formats.html#myst-markdown) that lets you move (fairly) seamlessly between `ipynb` and `md` files (of various flavours, including `myst`). 

I mention this as then the `code-cell` items are directives in the text file which gives you access to [all the cell metadata](https://myst-nb.readthedocs.io/en/latest/authoring/text-notebooks.html#cell-level-metadata) just like any other directive in a text editing environment, such as:

````md
```{code-cell} python
---
tags: [hide-output]
---
<code>
```
````

[Here is an example](https://raw.githubusercontent.com/QuantEcon/lecture-python.myst/main/lectures/aiyagari.md) md file if you're interested in that workflow.

Not sure if that helps, for your workflow choices. It would be nice to have more convenient access to tags in the vscode environment.

**Reply by agoose77** [reply]
Just an update: `jupyterlab-myst>=0.1.7a4` has reasonable support for MyST markdown rendering. It's not perfect, but if you want to use JupyterLab you're welcome to give it a try. It relies on the features that @rowanc1 and @chrisjsewell et al. have been working on.
